tests/unittest_regrtest.py

```diff
@@ -81,6 +81,19 @@ def test_numpy_crash(self):
         inferred = callfunc.inferred()
         self.assertEqual(len(inferred), 1)
 
+    @unittest.skipUnless(HAS_NUMPY, "Needs numpy")
+    def test_numpy_distutils(self):
+        """Special handling of virtualenv's patching of distutils shouldn't interfere
+        with numpy.distutils"""
```

Comment:
Do we need to define a specific version of numpy here ? I suppose numpy won't embark distutils forever.